documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -437,7 +437,7 @@ Debezium and Kafka Connect are designed around _continuous streams of event mess
 [[change-events-key]]
 ==== Change Event's Key
 
-For a given table, the change event's key will have a structure that contains a field for each column in the primary key (or unique key constraint) of the table at the time the event was created.
+For a given table, the change event's key will have a structure that contains a field for each column in the primary key (or unique key constraint with `REPLICA IDENTITY USING INDEX` set on the table) of the table at the time the event was created.
```

Comment:
Wouldn't a unique key work with RI `FULL` too?

Reply:
@gunnarmorling Of course it will. You are right that the sentence is confusing. I've added FULL option too